lib/sidekiq_publisher/datadog_apm.rb

```diff
@@ -43,6 +43,10 @@ def finish_span(payload)
       def service
         SidekiqPublisher::DatadogAPM.service
       end
+
+      def tracing
+        defined?(Datadog::Tracing) ? Datadog::Tracing : Datadog.tracer
```

Comment:
Could we extract this out into either a class method since we repeat this a couple of times in here and in specs

Reply:
Good call, done in `SidekiqPublisher::DatadogAPM.tracer`.